samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```diff
@@ -213,32 +226,33 @@ public void onJobModelExpired() {
           if (SamzaContainerStatus.NOT_STARTED.equals(status) || SamzaContainerStatus.STARTED.equals(status)) {
             boolean shutdownComplete = false;
             try {
-              LOGGER.info("Shutting down container in onJobModelExpired for processor:" + processorId);
+              LOGGER.info("Shutting down the container: {} of stream processor: {} in onJobModelExpired.", container, processorId);
               container.pause();
               shutdownComplete = jcContainerShutdownLatch.await(taskShutdownMs, TimeUnit.MILLISECONDS);
-              LOGGER.info("ShutdownComplete=" + shutdownComplete);
+              LOGGER.info("Container shutdown completion status is: {} for stream processor: {}.", shutdownComplete, processorId);
             } catch (IllegalContainerStateException icse) {
               // Ignored since container is not running
-              LOGGER.info("Container was not running.", icse);
+              LOGGER.info(String.format("Container: %s of stream processor: %s is dead. Shutdown is NO-OP.", container, processorId), icse);
```

Comment:
This is not a no-op, since we set shutdownComplete = true. Also don't guess that its dead.
"Container ... for processsor ... is in an illegal state. Marking container shutdown as complete."